OpenKh.WinShell.IMDUtilities/OpenKh.WinShell.IMDUtilities.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <RootNamespace>OpenKh.WinShell.IMDUtilities</RootNamespace>
+    <AssemblyName>OpenKh.WinShell.IMDUtilities</AssemblyName>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
Is the `Condition` valid only for Release and only for AnyCPU wanted? Is it even required to output the assembly to `OpenKh.WinShell.Output`?

Reply:
Well, the reason for the output being WinShell.Output is to prevent massive DLL copying and avoiding many undeletable and duplicate services in installation. But I will edit the condition to be valid at all costs. I somehow missed that, whoops!